### DISCUSSION TITLE
How to filter just multimidia created with the evidence (a cellphone)

### DISCUSSION THREAD

**Discussion by Klarissa-ksj** [sepinf-inc/IPED#408] [Q&A]
I'd like to select images, videos and audio records created with the cellphone that is my evidence, an iPhone 11.

In case of videos, I filtered by "ufed:camera make: apple".
For images may I filtered by "image:Make: Apple". 

For audio records I didn't find a value to filter.

**Comment by lfcnassif** [comment]
For images it is best to use image:Model metadata. For audios and videos, there isn't a standard like EXIF, the camera can or cannot save its model/serial info in the audio or video metadata, and each manufacturer can use a different metadata name, you should take a look at all audio/video metadata to see if device info is stored.

**Reply by wladimirleite** [reply]
@lfcnassif, as you said, this is highly dependent on the device and file format used.
I checked here a few iPhone UFDRs and realized that some audios do have interesting properties:
```
"Major Brand": "Apple iTunes"
"User Name": <the actual username>
"Handler Vendor ID": "Apple"
```
I found these values using external tools (mplayer and mostly exiftool), as they weren't extracted by IPED's audio parser.
I think it would be a possible (and interesting) enhancement extracting more properties from audio files.

As far as I know, TIKA standard parsers are used for audios. 
Maybe newer versions of such parsers already extract more information.

**Reply by lfcnassif** [reply]
I took a look at commit history of Tika audio and mp3 packages, nothing relevant in last 3 years...
Also vorbis plugin (for ogg, vorbis, flac, opus, speex...) last release was in 2016...

Maybe an [ExternalParser](https://github.com/sepinf-inc/IPED/blob/master/iped-app/resources/config/profiles/en/default/conf/ExternalParsers.xml) could be used for that? ffmpeg and exiftool examples are commented out.

**Reply by wladimirleite** [reply]
I see. Well, an external parser seems to be a good solution. There is a wide variety of audio formats being used nowadays, so it would require a deeper analysis to find the best alternative.

**Comment by Klarissa-ksj** [comment]
thank you all!
